apps/web/src/modules/proposal/components/ProposalVotes/VoterParticipation.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Text } from '@zoralabs/zord'
+import { useMemo } from 'react'
+
+import { useLayoutStore } from 'src/stores'
+
+import { VoterParticipationVariants } from './VoterParticipation.css'
+
+export interface VoterParticipationProps {
+  totalVotes: number
+  maxVotes: number
+}
+
+export const VoterParticipation: React.FC<VoterParticipationProps> = ({
+  totalVotes,
+  maxVotes,
+}) => {
+  const { isMobile } = useLayoutStore()
+  const participation = (totalVotes / maxVotes) * 100
+
+  const variant = useMemo(() => {
+    if (participation >= 65) return 'positive'
+    if (participation <= 25) return 'negative'
+    return 'neutral'
+  }, [participation])
+
+  return (
+    <Box
+      borderStyle="solid"
+      borderColor="border"
+      borderRadius="curved"
+      w="100%"
+      px={isMobile ? 'x4' : 'x6'}
+      py="x4"
+    >
+      <Text variant="heading-xs" style={{ fontWeight: 600 }}>
```

Comment:
nit: is `fontWeight={display}` not valid here?

Reply:
just switched it to `fontWeight={label}` which should be 600. 

sidenote I think theres a bug with the font weight properties on `Text` elements. `fontWeight={display}` should be set as 700 from what I'm seeing in the config but it renders as 500 for me. In general I find the font weight abstraction to be a bit confusing as its not immediately obvious to me if things like header or label should be darker.